Compatibility with Python 3.9

Hi there.

This is just a quick note. When running `make html` within `docker run -it --rm python:3.9.0 bash`, it errors out with
```
Installing collected packages: urllib3, pytz, pyparsing, MarkupSafe, idna, chardet, certifi, tornado, sphinxcontrib-serializinghtml, sphinxcontrib-qthelp, sphinxcontrib-jsmath, sphinxcontrib-htmlhelp, sphinxcontrib-devhelp, sphinxcontrib-applehelp, snowballstemmer, six, setuptools, requests, Pygments, packaging, Jinja2, imagesize, docutils, babel, alabaster, sphinx, livereload, sphinx-autobuild
    Running setup.py install for MarkupSafe ... done
  Attempting uninstall: setuptools
    Found existing installation: setuptools 49.2.1
    Uninstalling setuptools-49.2.1:
      Successfully uninstalled setuptools-49.2.1
    Running setup.py install for livereload ... error
    ERROR: Command errored out with exit status 1:
     command: /home/crate-docs/docs/.crate-docs/.venv/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-jrn94hws/livereload_30ab7443dd23497e995991957799188a/setup.py'"'"'; __file__='"'"'/tmp/pip-install-jrn94hws/livereload_30ab7443dd23497e995991957799188a/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-9bgqgv7d/install-record.txt --single-version-externally-managed --compile --install-headers /home/crate-docs/docs/.crate-docs/.venv/include/site/python3.9/livereload
         cwd: /tmp/pip-install-jrn94hws/livereload_30ab7443dd23497e995991957799188a/
    Complete output (11 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/home/crate-docs/docs/.crate-docs/.venv/lib/python3.9/site-packages/setuptools/__init__.py", line 20, in <module>
        from setuptools.dist import Distribution, Feature
      File "/home/crate-docs/docs/.crate-docs/.venv/lib/python3.9/site-packages/setuptools/dist.py", line 35, in <module>
        from setuptools.depends import Require
      File "/home/crate-docs/docs/.crate-docs/.venv/lib/python3.9/site-packages/setuptools/depends.py", line 7, in <module>
        from .py33compat import Bytecode
      File "/home/crate-docs/docs/.crate-docs/.venv/lib/python3.9/site-packages/setuptools/py33compat.py", line 55, in <module>
        unescape = getattr(html, 'unescape', html_parser.HTMLParser().unescape)
    AttributeError: 'HTMLParser' object has no attribute 'unescape'
    ----------------------------------------
ERROR: Command errored out with exit status 1: /home/crate-docs/docs/.crate-docs/.venv/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-jrn94hws/livereload_30ab7443dd23497e995991957799188a/setup.py'"'"'; __file__='"'"'/tmp/pip-install-jrn94hws/livereload_30ab7443dd23497e995991957799188a/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-9bgqgv7d/install-record.txt --single-version-externally-managed --compile --install-headers /home/crate-docs/docs/.crate-docs/.venv/include/site/python3.9/livereload Check the logs for full command output.
make[1]: *** [.crate-docs/src/rules.mk:110: .crate-docs/.venv/bin/sphinx-build] Error 1
make[1]: Leaving directory '/home/crate-docs/docs'
make: *** [Makefile:69: html] Error 2
```

With kind regards,
Andreas.

--- a/.github/workflows/docs.yml
+++ b/.github/workflows/docs.yml
@@ -7,19 +7,20 @@ on:
 
 jobs:
   documentation:
-    name: Build docs on ${{ matrix.os }}
+    name: Build docs on ${{ matrix.os }} using Python ${{ matrix.python-version }}
     runs-on: ${{ matrix.os }}
 
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest]
+        python-version: ["3.7", "3.8", "3.9"]
 
     steps:
       - uses: actions/checkout@v2
       - name: Set up Python
         uses: actions/setup-python@v1
         with:
-          python-version: 3.7
+          python-version: ${{ matrix.python-version }}
       - name: Build docs
         run: |
           cd docs && make check

--- a/src/requirements.txt
+++ b/src/requirements.txt
@@ -6,5 +6,5 @@ sphinx-autobuild
 
 # These versions should match the Read The Docs environment (please update if
 # you notice this is not the case)
-setuptools<41
+setuptools>=41.1.0
 sphinx>=1.8.5,<4

